packages/suite/src/components/suite/modals/AddAccount/components/NetworkSelect.tsx

```diff
@@ -48,6 +48,21 @@ const NetworkOption = ({ value, label }: Option) => (
     </OptionWrapper>
 );
 
+const customNetworkFilter = (option: Option, searchText: string) => {
+    const searchString = searchText.toLowerCase();
+
+    const matchLabel = option.label.toLowerCase().includes(searchString);
+    const matchSymbol = option.value.symbol.toLowerCase().includes(searchString);
+    // match XRP/TXRP when user types "Ripple"
+    const matchXRPAlternativeName =
+        option.value.networkType === 'ripple' && 'ripple'.includes(searchString);
+
+    if (matchLabel || matchSymbol || matchXRPAlternativeName) {
+        return true;
+    }
+    return false;
```

Comment:
You could actually just return the result from the condition: `return matchLabel || matchSymbol || matchXRPAlternativeName;`

Reply:
ahh ofc, good point!